docs/policygenerator-reference.yaml

```diff
@@ -108,11 +108,14 @@ policyDefaults:
   # Optional. The placement configuration for the policies. This defaults to a placement configuration that matches all
   # clusters.
   placement:
-    # To specify a placement rule, specify key:value pair cluster selectors. (See placementRulePath to specify an
-    # existing file instead.)
+    # To specify a placement rule, specify key:value pair cluster selectors or the full YAML for the desired cluster
+    # selectors. (See placementRulePath to specify an existing file instead.)
     clusterSelectors: {}
```

Comment:
Do you think we could just remove this from the docs so that new users start using `clusterSelector`?

Reply:
I think it might be a bit confusing to existing users if we completely remove it - what if we just added a recommendation to use clusterSelector here, similar to the error message?

Reply:
Sounds good!